fury/window.py

```diff
@@ -434,6 +440,11 @@ def start(self):
         del self.iren
         del self.window
 
+    @property
+    def frame_rate(self):
```

Comment:
Old definition of frame_rate based on GetLastRenderTimeInSeconds was not removed.

Reply:
Since it's being used in some tests, and it's already getting calculated anyway, we may just change its name and return what "GetLastRenderTimeInSeconds" returns. It would be good to have a benchmark for the rendering itself.